tests/app/libs/test_utils.py

```diff
@@ -1,20 +1,29 @@
-import unittest
+import pytest
 
-from app.libs.utils import ObjectFromDict
+from app.helpers.uuid_helper import is_valid_uuid
+from app.libs.utils import convert_tx_id, escape_value
 
 
-class ObjectFromDictTest(unittest.TestCase):
-    def test_simple_object(self):
-        simple_dict = {
-            "property_one": "string",
-            "property_two": 2,
-            "property_three": [],
-        }
+def test_convert_tx_id():
```

Comment:
slightly confused why this has been added

Reply:
It was a method in `utils.py` which I added type hints to as I introduced type hints and changes to that file. Therefore, it made sense to add a test for it while I was adding tests for the rest of the file,  since it wasn't tested as a unit.